runtime/runtime_test.go

```diff
@@ -5736,7 +5734,7 @@ func TestRuntimeStorageWriteback(t *testing.T) {
 			// resource value
 			{
 				addressValue[:],
-				[]byte{'$', 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x4},
+				[]byte{'$', 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x3},
```

Comment:
The storage map now has ID 4, as it is now created after the resource, see the change of order of writes above